algorithms/ar-lscsa/typescript/largest_sum_contiguous_subarray.ts

```diff
@@ -0,0 +1,24 @@
+function largestSumContiguousSubarray (arr: number[]) {
+    let globalMax = Number.MIN_VALUE;
+    let localMax = 0;
+
+    for (const num of arr) {
+        localMax += num;
+
+        if (globalMax < localMax) {
+            globalMax = localMax;
+        }
+        if (localMax < 0) {
```

Comment:
Codacy found an issue: [Irregular whitespace not allowed.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6391064)